tests/test_api.py

```diff
@@ -29,6 +29,16 @@ def test_word_search_no_results(self):
         resp_json = resp.json()
         self.assertEqual(0, resp_json["count"])
 
+class LexiconAPITestCase(TestCase):
+    fixtures = ['lexicon-sample.json']
+
+    def test_lexicon(self):
+        resp = self.client.get('/api/lexicon/get_lexicon_names/')
```

Comment:
Standar API REST will expects that URL was `/api/lexicon/` OR better in plural `/api/lexicons/`.

Reply:
API refactored to use the default method and avoid using the overwritten one, also changed the default method name from lexicon to lexicons as you requested